docs/appendices/comparison-to-related-tools.md

```diff
@@ -138,6 +138,26 @@ there are some major limitations:
         not every CloudTrail action matches IAM Actions, the results are
         not always accurate.
 
+Monitoring AWS API Calls
```

Comment:
This is great! Updated comparisons to existing tools is very overdue. Really appreciate it.

Reply:
Added `iamlive` in the comparison docs as well, let me know what you think!

Also let me know if there's any tools you'd like to add here that you haven't had time to, I might have some cycles to spare on adding those to this doc